### ISSUE TITLE
Weight normalization

### ISSUE THREAD

**Issue by theFilipko** [NVlabs/tiny-cuda-nn#59]
Hello. I would like to ask, do you plan to implement the weight normalization? Thank you.

**Comment by Tom94** [comment]
Hi there, while there's currently no high-priority plan to add batch normalization (or other weight normalization schemes), this is something that would be cool to have in the future.

I'm leaving this issue open as a TODO marker.

Cheers!

**Comment by jingyibo123** [comment]
Effectively the recent [Zip-NeRF](https://arxiv.org/abs/2304.06706) mentioned a `normalized weight decay` which greatly improves performance.
